internal/venice-test-common/src/integrationTest/java/com/linkedin/venice/fastclient/AvroStoreClientGzipEndToEndTest.java

```diff
@@ -58,12 +57,11 @@ public Object[][] fastClientTestPermutations() {
         DataProviderUtils.BOOLEAN, // enableGrpc
         DataProviderUtils.BOOLEAN, // retryEnabled
         BATCH_GET_KEY_SIZE.toArray(), // batchGetKeySize
-        REQUEST_TYPES_SMALL, // requestType
-        STORE_METADATA_FETCH_MODES); // storeMetadataFetchMode
+        REQUEST_TYPES_SMALL); // requestType
```

Comment:
This needs to be removed from this and its overridden functions. Removing `StoreMetadataFetchMode` as per Sushant's comments will take care of this as well, but just putting it out here just in case if that is not done.
`
StoreMetadataFetchMode storeMetadataFetchMode = (StoreMetadataFetchMode) permutation[7];
`